### ISSUE TITLE
intermitted nodeos_forked_chain_lr_test failure in CICD

### ISSUE THREAD

**Issue by linh2931** [eosnetworkfoundation/mandel#490]
In my merge build, `nodeos_forked_chain_lr_test` failed:

```
2022-06-21T18:54:51.923425 blockNum = 686 blockProducer0 = defproducerl blockProducer1 = defproducerl
2022-06-21T18:54:51.923490 nextProdChange = False
2022-06-21T18:54:51.923550 missedTransitionBlock = 686 transitionCount = 2
2022-06-21T18:54:51.923609 At block 686 and have passed producer: defproducerk 2 times and we have not diverged, stopping looking and letting errors report
2022-06-21T18:54:51.923709  Checking if node(pid=64) is alive(killed=False): programs/nodeos/nodeos --resource-monitor-not-shutdown-on-threshold-exceeded --max-transaction-time -1 --abi-serializer-max-time-ms 990000 --p2p-max-nodes-per-host 3 --contracts-console --plugin eosio::producer_api_plugin --plugin eosio::trace_api_plugin --trace-no-abis --http-max-response-time-ms 990000 --plugin eosio::test_control_api_plugin --config-dir etc/eosio/node_02 --data-dir var/lib/node_02 --genesis-json etc/eosio/node_02/genesis.json --genesis-timestamp 2022-06-21T18:49:05.764
2022-06-21T18:54:51.923830  Determined node(pid=64) is alive
2022-06-21T18:54:51.923922  ERROR: Expected the non-producing node to have shutdown.
2022-06-21T18:54:51.924169  Test failed.
```

Please see 
[log.txt](https://github.com/eosnetworkfoundation/mandel/files/8952959/log.txt)
for details.

https://github.com/EOSIO/eos/pull/9007 has already been backported. It did not help; something else went wrong.

**Comment by linh2931** [comment]
This might be related to https://github.com/eosnetworkfoundation/mandel/issues/405 and https://github.com/eosnetworkfoundation/mandel/issues/437. Let me backport them first and see if the problem still happens.

**Comment by linh2931** [comment]
This is not related to https://github.com/eosnetworkfoundation/mandel/issues/405 and https://github.com/eosnetworkfoundation/mandel/issues/437. The Python log shows `kill_node_on_producer` command was sent to the bridge node's RPC endpoint. Without `nodeds` logs, I cannot tell what had happened after the bridge node received the command.

I am trying to reproduce the problem locally but so far without any luck.

**Comment by linh2931** [comment]
Blocked by https://github.com/eosnetworkfoundation/mandel/issues/513

**Comment by spoonincode** [comment]
Considering this closed via AntelopeIO/leap#667
